db/clean_planting_csv/clean_planting_API_call.py

```diff
@@ -0,0 +1,97 @@
+import requests, zipfile, io
+import json
+import pandas as pd
+import uuid
+
+central_url = "https://senecaparkzoo.getodk.cloud"
+central_email = "email"
+central_password = "yourpassword"
+
+#gets authentication token
+def get_email_token():
+
+    email_token_response = requests.post(
+        central_url + "/v1/sessions",
+        data=json.dumps({"email": central_email, "password": central_password}),
+        headers={"Content-Type": "application/json"},
+    )
+
+    if email_token_response.status_code == 200:
```

Comment:
If something happens in your program that essentially means it cannot continue unless handled, you can `raise` an exception. So if you need an email_token to finish the script and the response isn't a 200, you can have an `else` statement where you `raise Exception("Invalid email response")` or something like that. That will interrupt the program and prevent the rest of it from running and breaking later on

Reply:
Ok, will make that change!

Reply:
I should also do this for the block that checks for required columns.